modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLCopyEntryDisplayContext.java

```diff
@@ -56,20 +58,14 @@ public DLCopyEntryDisplayContext(
 		_themeDisplay = themeDisplay;
 	}
 
-	public String getActionURL() {
-		String actionName = "/document_library/copy_file_entry";
-
-		if (getFileShortcutId() > 0) {
-			actionName = "/document_library/copy_file_shortcut";
+	public String getCopyToURL() {
+		if (_copyEntryURL != null) {
+			return _copyEntryURL;
 		}
 
-		return PortletURLBuilder.createActionURL(
-			_liferayPortletResponse
-		).setActionName(
-			actionName
-		).setMVCRenderCommandName(
-			actionName
-		).buildString();
+		return StringBundler.concat(
+			PortalUtil.getPortalURL(_httpServletRequest),
+			PortalUtil.getPathContext(), Portal.PATH_MODULE, "/copy_dl_entry");
```

Comment:
The other day, Brian asked me to follow the "kbObject" pattern used in KB. Can you rename the URL/Servlet accordingly? "copy_dl_entry" -> "copy_dl_object".